.github/workflows/reusable-image-scan-and-release-ghcr.yml

```diff
@@ -84,7 +84,7 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
       - name: Login to Docker
-        uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.4.0
```

Comment:
Stemmer: https://github.com/Altinn/altinn-platform/pull/1456/files#diff-0b86917293b396a7379a4c1cbb51a823c6bd8ce5a98db360fccd925ec0877674L87.
Did you look through the rest to see if there were more mess ups?

Reply:
Scrolled through the original PR, where I messed up a suggestion, and couldn't find any other places where the action was changed. [Orig PR](https://github.com/Altinn/altinn-platform/pull/1456/files)